src/sonic-yang-models/yang-templates/sonic-types.yang.j2

```diff
@@ -229,9 +229,14 @@ module sonic-types {
         }
     }
 
+    typedef hostname {
+        type string {
+            length 1..255;
```

Comment:
Not sure about this limit, why 255?
HOST_NAME_MAX is 64 by default.

Reply:
This seems to be the default length given for other string types in the YANG model. I can change to 64 here.

Reply:
> This seems to be the default length given for other string types in the YANG model. I can change to 64 here.

I'm not sure, need to confirm

Reply:
Resolved offline, according to [RFC1035](https://www.rfc-editor.org/rfc/rfc1035#:~:text=more%0Afundamental.%0A%0Alabels-,63%20octets%20or%20less,-names%20%20%20%20%20%20%20%20%20%20%20255%20octets) the max length for a label is 63 characters.